modules/swagger-codegen/src/main/resources/aspnet5/model.mustache

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Linq;
+using System.IO;
+using System.Text;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+{{#models}}
+{{#model}}
+namespace {{packageName}}.Models
+{
+    /// <summary>
+    /// {{description}}
+    /// </summary>
+    public class {{classname}} : {{#parent}}{{{parent}}}, {{/parent}} IEquatable<{{classname}}>
```

Comment:
[OPTIONAL] we just added `partial` to the model class. You might want to do the same.